Municipality/azc.mod2g

```diff
@@ -0,0 +1,13 @@
+use dummy as knowledge.
+
+order = linearall.
+module azc{
+	%%% check if there is still need for an azc (id 650)
+	if bel(building(Id,_,OwnerID,_,_,650,_)) then exit-module.
+	%%% check if we have budget for an azc
+	if bel(money(X),X<100000) then exit-module.
```

Comment:
Let comments start with a capital letter for consistency (one of the current issues from our Code Evolution Quality)